src/expressions/match-expr.md

```diff
@@ -95,18 +95,23 @@ match x {
 }
 ```
 
-Multiple match patterns may be joined with the `|` operator. A range of values
-may be specified with `...`. For example:
+Multiple match patterns may be joined with the `|` operator. An inclusive range
+of values may be specified with `..=`. For example:
 
 ```rust
-# let x = 2;
+# let x = 9;
 let message = match x {
     0 | 1  => "not many",
-    2 ... 9 => "a few",
+    2 ..= 9 => "a few",
     _      => "lots"
 };
+
+assert_eq!(message, "a few");
 ```
 
+Other forms of [range] \(`..` for an exclusive range, or any range with one or
+both endpoints left unspecified) are not currently supported in matches.
```

Comment:
I don't think that 'currently' is needed, the reference only talks about Rust at this point in time.

Reply:
Done. I also remarked that `...` is still accepted for backwards compatibility.